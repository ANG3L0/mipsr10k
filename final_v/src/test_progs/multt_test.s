
        data1 = 0x0001   
        data2 = 0x000a
        data3 = 0x0002
        data4 = 0x0008

        lda  $r1, data1   /* $r1 = 1 */
        lda  $r2, data2   /* $r2 = 2 */
        lda  $r3, data3   /* $r3 = 6 */
        lda  $r4, data4   /* $r4 = 8 */
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
        mulq $r3, $r3, $r3
#       mult $r, $r1, $r2
        call_pal 0x555
