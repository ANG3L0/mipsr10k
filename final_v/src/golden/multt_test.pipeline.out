                                                                            D-MEM Bus &
Cycle:      IF      |     ID      |     EX      |     MEM     |     WB      Reg Result
    0:    4:-       |   0:-       |   0:-       |   0:-       |   0:-       
    1:    4:-       |   4:-       |   0:-       |   0:-       |   0:-       BUS_LOAD  MEM[0] accepted 1
    2:    4:-       |   4:-       |   4:-       |   0:-       |   0:-       
    3:    4:-       |   4:-       |   4:-       |   4:-       |   0:-       
    4:    4:-       |   4:-       |   4:-       |   4:-       |   4:-       
    5:    4:-       |   4:-       |   4:-       |   4:-       |   4:-       
    6:    4:lda     |   4:-       |   4:-       |   4:-       |   4:-       
    7:    8:-       |   4:lda     |   4:-       |   4:-       |   4:-       
    8:    8:-       |   8:-       |   4:lda     |   4:-       |   4:-       
    9:    8:-       |   8:-       |   8:-       |   4:lda     |   4:-       
   10:    8:-       |   8:-       |   8:-       |   8:-       |   4:lda     r1=1  
   11:    8:lda     |   8:-       |   8:-       |   8:-       |   8:-       
   12:   12:-       |   8:lda     |   8:-       |   8:-       |   8:-       
   13:   12:-       |  12:-       |   8:lda     |   8:-       |   8:-       BUS_LOAD  MEM[8] accepted 1
   14:   12:-       |  12:-       |  12:-       |   8:lda     |   8:-       
   15:   12:-       |  12:-       |  12:-       |  12:-       |   8:lda     r2=10  
   16:   12:-       |  12:-       |  12:-       |  12:-       |  12:-       
   17:   12:-       |  12:-       |  12:-       |  12:-       |  12:-       
   18:   12:lda     |  12:-       |  12:-       |  12:-       |  12:-       
   19:   16:-       |  12:lda     |  12:-       |  12:-       |  12:-       
   20:   16:-       |  16:-       |  12:lda     |  12:-       |  12:-       
   21:   16:-       |  16:-       |  16:-       |  12:lda     |  12:-       
   22:   16:-       |  16:-       |  16:-       |  16:-       |  12:lda     r3=2  
   23:   16:lda     |  16:-       |  16:-       |  16:-       |  16:-       
   24:   20:-       |  16:lda     |  16:-       |  16:-       |  16:-       
   25:   20:-       |  20:-       |  16:lda     |  16:-       |  16:-       BUS_LOAD  MEM[16] accepted 1
   26:   20:-       |  20:-       |  20:-       |  16:lda     |  16:-       
   27:   20:-       |  20:-       |  20:-       |  20:-       |  16:lda     r4=8  
   28:   20:-       |  20:-       |  20:-       |  20:-       |  20:-       
   29:   20:-       |  20:-       |  20:-       |  20:-       |  20:-       
   30:   20:mulq    |  20:-       |  20:-       |  20:-       |  20:-       
   31:   24:-       |  20:mulq    |  20:-       |  20:-       |  20:-       
   32:   24:-       |  24:-       |  20:mulq    |  20:-       |  20:-       
   33:   24:-       |  24:-       |  24:-       |  20:mulq    |  20:-       
   34:   24:-       |  24:-       |  24:-       |  24:-       |  20:mulq    r3=4  
   35:   24:mulq    |  24:-       |  24:-       |  24:-       |  24:-       
   36:   28:-       |  24:mulq    |  24:-       |  24:-       |  24:-       
   37:   28:-       |  28:-       |  24:mulq    |  24:-       |  24:-       BUS_LOAD  MEM[24] accepted 1
   38:   28:-       |  28:-       |  28:-       |  24:mulq    |  24:-       
   39:   28:-       |  28:-       |  28:-       |  28:-       |  24:mulq    r3=16  
   40:   28:-       |  28:-       |  28:-       |  28:-       |  28:-       
   41:   28:-       |  28:-       |  28:-       |  28:-       |  28:-       
   42:   28:mulq    |  28:-       |  28:-       |  28:-       |  28:-       
   43:   32:-       |  28:mulq    |  28:-       |  28:-       |  28:-       
   44:   32:-       |  32:-       |  28:mulq    |  28:-       |  28:-       
   45:   32:-       |  32:-       |  32:-       |  28:mulq    |  28:-       
   46:   32:-       |  32:-       |  32:-       |  32:-       |  28:mulq    r3=256  
   47:   32:mulq    |  32:-       |  32:-       |  32:-       |  32:-       
   48:   36:-       |  32:mulq    |  32:-       |  32:-       |  32:-       
   49:   36:-       |  36:-       |  32:mulq    |  32:-       |  32:-       BUS_LOAD  MEM[32] accepted 1
   50:   36:-       |  36:-       |  36:-       |  32:mulq    |  32:-       
   51:   36:-       |  36:-       |  36:-       |  36:-       |  32:mulq    r3=65536  
   52:   36:-       |  36:-       |  36:-       |  36:-       |  36:-       
   53:   36:-       |  36:-       |  36:-       |  36:-       |  36:-       
   54:   36:mulq    |  36:-       |  36:-       |  36:-       |  36:-       
   55:   40:-       |  36:mulq    |  36:-       |  36:-       |  36:-       
   56:   40:-       |  40:-       |  36:mulq    |  36:-       |  36:-       
   57:   40:-       |  40:-       |  40:-       |  36:mulq    |  36:-       
   58:   40:-       |  40:-       |  40:-       |  40:-       |  36:mulq    r3=0x10  
   59:   40:mulq    |  40:-       |  40:-       |  40:-       |  40:-       
   60:   44:-       |  40:mulq    |  40:-       |  40:-       |  40:-       
   61:   44:-       |  44:-       |  40:mulq    |  40:-       |  40:-       BUS_LOAD  MEM[40] accepted 1
   62:   44:-       |  44:-       |  44:-       |  40:mulq    |  40:-       
   63:   44:-       |  44:-       |  44:-       |  44:-       |  40:mulq    r3=0  
   64:   44:-       |  44:-       |  44:-       |  44:-       |  44:-       
   65:   44:-       |  44:-       |  44:-       |  44:-       |  44:-       
   66:   44:mulq    |  44:-       |  44:-       |  44:-       |  44:-       
   67:   48:-       |  44:mulq    |  44:-       |  44:-       |  44:-       
   68:   48:-       |  48:-       |  44:mulq    |  44:-       |  44:-       
   69:   48:-       |  48:-       |  48:-       |  44:mulq    |  44:-       
   70:   48:-       |  48:-       |  48:-       |  48:-       |  44:mulq    r3=0  
   71:   48:mulq    |  48:-       |  48:-       |  48:-       |  48:-       
   72:   52:-       |  48:mulq    |  48:-       |  48:-       |  48:-       
   73:   52:-       |  52:-       |  48:mulq    |  48:-       |  48:-       BUS_LOAD  MEM[48] accepted 1
   74:   52:-       |  52:-       |  52:-       |  48:mulq    |  48:-       
   75:   52:-       |  52:-       |  52:-       |  52:-       |  48:mulq    r3=0  
   76:   52:-       |  52:-       |  52:-       |  52:-       |  52:-       
   77:   52:-       |  52:-       |  52:-       |  52:-       |  52:-       
   78:   52:mulq    |  52:-       |  52:-       |  52:-       |  52:-       
   79:   56:-       |  52:mulq    |  52:-       |  52:-       |  52:-       
   80:   56:-       |  56:-       |  52:mulq    |  52:-       |  52:-       
   81:   56:-       |  56:-       |  56:-       |  52:mulq    |  52:-       
   82:   56:-       |  56:-       |  56:-       |  56:-       |  52:mulq    r3=0  
   83:   56:mulq    |  56:-       |  56:-       |  56:-       |  56:-       
   84:   60:-       |  56:mulq    |  56:-       |  56:-       |  56:-       
   85:   60:-       |  60:-       |  56:mulq    |  56:-       |  56:-       BUS_LOAD  MEM[56] accepted 1
   86:   60:-       |  60:-       |  60:-       |  56:mulq    |  56:-       
   87:   60:-       |  60:-       |  60:-       |  60:-       |  56:mulq    r3=0  
   88:   60:-       |  60:-       |  60:-       |  60:-       |  60:-       
   89:   60:-       |  60:-       |  60:-       |  60:-       |  60:-       
   90:   60:mulq    |  60:-       |  60:-       |  60:-       |  60:-       
   91:   64:-       |  60:mulq    |  60:-       |  60:-       |  60:-       
   92:   64:-       |  64:-       |  60:mulq    |  60:-       |  60:-       
   93:   64:-       |  64:-       |  64:-       |  60:mulq    |  60:-       
   94:   64:-       |  64:-       |  64:-       |  64:-       |  60:mulq    r3=0  
   95:   64:mulq    |  64:-       |  64:-       |  64:-       |  64:-       
   96:   68:-       |  64:mulq    |  64:-       |  64:-       |  64:-       
   97:   68:-       |  68:-       |  64:mulq    |  64:-       |  64:-       BUS_LOAD  MEM[64] accepted 1
   98:   68:-       |  68:-       |  68:-       |  64:mulq    |  64:-       
   99:   68:-       |  68:-       |  68:-       |  68:-       |  64:mulq    r3=0  
  100:   68:-       |  68:-       |  68:-       |  68:-       |  68:-       
  101:   68:-       |  68:-       |  68:-       |  68:-       |  68:-       
  102:   68:mulq    |  68:-       |  68:-       |  68:-       |  68:-       
  103:   72:-       |  68:mulq    |  68:-       |  68:-       |  68:-       
  104:   72:-       |  72:-       |  68:mulq    |  68:-       |  68:-       
  105:   72:-       |  72:-       |  72:-       |  68:mulq    |  68:-       
  106:   72:-       |  72:-       |  72:-       |  72:-       |  68:mulq    r3=0  
  107:   72:mulq    |  72:-       |  72:-       |  72:-       |  72:-       
  108:   76:-       |  72:mulq    |  72:-       |  72:-       |  72:-       
  109:   76:-       |  76:-       |  72:mulq    |  72:-       |  72:-       BUS_LOAD  MEM[72] accepted 1
  110:   76:-       |  76:-       |  76:-       |  72:mulq    |  72:-       
  111:   76:-       |  76:-       |  76:-       |  76:-       |  72:mulq    r3=0  
  112:   76:-       |  76:-       |  76:-       |  76:-       |  76:-       
  113:   76:-       |  76:-       |  76:-       |  76:-       |  76:-       
  114:   76:mulq    |  76:-       |  76:-       |  76:-       |  76:-       
  115:   80:-       |  76:mulq    |  76:-       |  76:-       |  76:-       
  116:   80:-       |  80:-       |  76:mulq    |  76:-       |  76:-       
  117:   80:-       |  80:-       |  80:-       |  76:mulq    |  76:-       
  118:   80:-       |  80:-       |  80:-       |  80:-       |  76:mulq    r3=0  
  119:   80:mulq    |  80:-       |  80:-       |  80:-       |  80:-       
  120:   84:-       |  80:mulq    |  80:-       |  80:-       |  80:-       
  121:   84:-       |  84:-       |  80:mulq    |  80:-       |  80:-       BUS_LOAD  MEM[80] accepted 1
  122:   84:-       |  84:-       |  84:-       |  80:mulq    |  80:-       
  123:   84:-       |  84:-       |  84:-       |  84:-       |  80:mulq    r3=0  
  124:   84:-       |  84:-       |  84:-       |  84:-       |  84:-       
  125:   84:-       |  84:-       |  84:-       |  84:-       |  84:-       
  126:   84:mulq    |  84:-       |  84:-       |  84:-       |  84:-       
  127:   88:-       |  84:mulq    |  84:-       |  84:-       |  84:-       
  128:   88:-       |  88:-       |  84:mulq    |  84:-       |  84:-       
  129:   88:-       |  88:-       |  88:-       |  84:mulq    |  84:-       
  130:   88:-       |  88:-       |  88:-       |  88:-       |  84:mulq    r3=0  
  131:   88:mulq    |  88:-       |  88:-       |  88:-       |  88:-       
  132:   92:-       |  88:mulq    |  88:-       |  88:-       |  88:-       
  133:   92:-       |  92:-       |  88:mulq    |  88:-       |  88:-       BUS_LOAD  MEM[88] accepted 1
  134:   92:-       |  92:-       |  92:-       |  88:mulq    |  88:-       
  135:   92:-       |  92:-       |  92:-       |  92:-       |  88:mulq    r3=0  
  136:   92:-       |  92:-       |  92:-       |  92:-       |  92:-       
  137:   92:-       |  92:-       |  92:-       |  92:-       |  92:-       
  138:   92:mulq    |  92:-       |  92:-       |  92:-       |  92:-       
  139:   96:-       |  92:mulq    |  92:-       |  92:-       |  92:-       
  140:   96:-       |  96:-       |  92:mulq    |  92:-       |  92:-       
  141:   96:-       |  96:-       |  96:-       |  92:mulq    |  92:-       
  142:   96:-       |  96:-       |  96:-       |  96:-       |  92:mulq    r3=0  
  143:   96:mulq    |  96:-       |  96:-       |  96:-       |  96:-       
  144:  100:-       |  96:mulq    |  96:-       |  96:-       |  96:-       
  145:  100:-       | 100:-       |  96:mulq    |  96:-       |  96:-       BUS_LOAD  MEM[96] accepted 1
  146:  100:-       | 100:-       | 100:-       |  96:mulq    |  96:-       
  147:  100:-       | 100:-       | 100:-       | 100:-       |  96:mulq    r3=0  
  148:  100:-       | 100:-       | 100:-       | 100:-       | 100:-       
  149:  100:-       | 100:-       | 100:-       | 100:-       | 100:-       
  150:  100:mulq    | 100:-       | 100:-       | 100:-       | 100:-       
  151:  104:-       | 100:mulq    | 100:-       | 100:-       | 100:-       
  152:  104:-       | 104:-       | 100:mulq    | 100:-       | 100:-       
  153:  104:-       | 104:-       | 104:-       | 100:mulq    | 100:-       
  154:  104:-       | 104:-       | 104:-       | 104:-       | 100:mulq    r3=0  
  155:  104:mulq    | 104:-       | 104:-       | 104:-       | 104:-       
  156:  108:-       | 104:mulq    | 104:-       | 104:-       | 104:-       
  157:  108:-       | 108:-       | 104:mulq    | 104:-       | 104:-       BUS_LOAD  MEM[104] accepted 1
  158:  108:-       | 108:-       | 108:-       | 104:mulq    | 104:-       
  159:  108:-       | 108:-       | 108:-       | 108:-       | 104:mulq    r3=0  
  160:  108:-       | 108:-       | 108:-       | 108:-       | 108:-       
  161:  108:-       | 108:-       | 108:-       | 108:-       | 108:-       
  162:  108:mulq    | 108:-       | 108:-       | 108:-       | 108:-       
  163:  112:-       | 108:mulq    | 108:-       | 108:-       | 108:-       
  164:  112:-       | 112:-       | 108:mulq    | 108:-       | 108:-       
  165:  112:-       | 112:-       | 112:-       | 108:mulq    | 108:-       
  166:  112:-       | 112:-       | 112:-       | 112:-       | 108:mulq    r3=0  
  167:  112:mulq    | 112:-       | 112:-       | 112:-       | 112:-       
  168:  116:-       | 112:mulq    | 112:-       | 112:-       | 112:-       
  169:  116:-       | 116:-       | 112:mulq    | 112:-       | 112:-       BUS_LOAD  MEM[112] accepted 1
  170:  116:-       | 116:-       | 116:-       | 112:mulq    | 112:-       
  171:  116:-       | 116:-       | 116:-       | 116:-       | 112:mulq    r3=0  
  172:  116:-       | 116:-       | 116:-       | 116:-       | 116:-       
  173:  116:-       | 116:-       | 116:-       | 116:-       | 116:-       
  174:  116:mulq    | 116:-       | 116:-       | 116:-       | 116:-       
  175:  120:-       | 116:mulq    | 116:-       | 116:-       | 116:-       
  176:  120:-       | 120:-       | 116:mulq    | 116:-       | 116:-       
  177:  120:-       | 120:-       | 120:-       | 116:mulq    | 116:-       
  178:  120:-       | 120:-       | 120:-       | 120:-       | 116:mulq    r3=0  
  179:  120:mulq    | 120:-       | 120:-       | 120:-       | 120:-       
  180:  124:-       | 120:mulq    | 120:-       | 120:-       | 120:-       
  181:  124:-       | 124:-       | 120:mulq    | 120:-       | 120:-       BUS_LOAD  MEM[120] accepted 1
  182:  124:-       | 124:-       | 124:-       | 120:mulq    | 120:-       
  183:  124:-       | 124:-       | 124:-       | 124:-       | 120:mulq    r3=0  
  184:  124:-       | 124:-       | 124:-       | 124:-       | 124:-       
  185:  124:-       | 124:-       | 124:-       | 124:-       | 124:-       
  186:  124:mulq    | 124:-       | 124:-       | 124:-       | 124:-       
  187:  128:-       | 124:mulq    | 124:-       | 124:-       | 124:-       
  188:  128:-       | 128:-       | 124:mulq    | 124:-       | 124:-       
  189:  128:-       | 128:-       | 128:-       | 124:mulq    | 124:-       
  190:  128:-       | 128:-       | 128:-       | 128:-       | 124:mulq    r3=0  
  191:  128:mulq    | 128:-       | 128:-       | 128:-       | 128:-       
  192:  132:-       | 128:mulq    | 128:-       | 128:-       | 128:-       
  193:  132:-       | 132:-       | 128:mulq    | 128:-       | 128:-       BUS_LOAD  MEM[128] accepted 1
  194:  132:-       | 132:-       | 132:-       | 128:mulq    | 128:-       
  195:  132:-       | 132:-       | 132:-       | 132:-       | 128:mulq    r3=0  
  196:  132:-       | 132:-       | 132:-       | 132:-       | 132:-       
  197:  132:-       | 132:-       | 132:-       | 132:-       | 132:-       
  198:  132:mulq    | 132:-       | 132:-       | 132:-       | 132:-       
  199:  136:-       | 132:mulq    | 132:-       | 132:-       | 132:-       
  200:  136:-       | 136:-       | 132:mulq    | 132:-       | 132:-       
  201:  136:-       | 136:-       | 136:-       | 132:mulq    | 132:-       
  202:  136:-       | 136:-       | 136:-       | 136:-       | 132:mulq    r3=0  
  203:  136:mulq    | 136:-       | 136:-       | 136:-       | 136:-       
  204:  140:-       | 136:mulq    | 136:-       | 136:-       | 136:-       
  205:  140:-       | 140:-       | 136:mulq    | 136:-       | 136:-       BUS_LOAD  MEM[136] accepted 1
  206:  140:-       | 140:-       | 140:-       | 136:mulq    | 136:-       
  207:  140:-       | 140:-       | 140:-       | 140:-       | 136:mulq    r3=0  
  208:  140:-       | 140:-       | 140:-       | 140:-       | 140:-       
  209:  140:-       | 140:-       | 140:-       | 140:-       | 140:-       
  210:  140:mulq    | 140:-       | 140:-       | 140:-       | 140:-       
  211:  144:-       | 140:mulq    | 140:-       | 140:-       | 140:-       
  212:  144:-       | 144:-       | 140:mulq    | 140:-       | 140:-       
  213:  144:-       | 144:-       | 144:-       | 140:mulq    | 140:-       
  214:  144:-       | 144:-       | 144:-       | 144:-       | 140:mulq    r3=0  
  215:  144:mulq    | 144:-       | 144:-       | 144:-       | 144:-       
  216:  148:-       | 144:mulq    | 144:-       | 144:-       | 144:-       
  217:  148:-       | 148:-       | 144:mulq    | 144:-       | 144:-       BUS_LOAD  MEM[144] accepted 1
  218:  148:-       | 148:-       | 148:-       | 144:mulq    | 144:-       
  219:  148:-       | 148:-       | 148:-       | 148:-       | 144:mulq    r3=0  
  220:  148:-       | 148:-       | 148:-       | 148:-       | 148:-       
  221:  148:-       | 148:-       | 148:-       | 148:-       | 148:-       
  222:  148:mulq    | 148:-       | 148:-       | 148:-       | 148:-       
  223:  152:-       | 148:mulq    | 148:-       | 148:-       | 148:-       
  224:  152:-       | 152:-       | 148:mulq    | 148:-       | 148:-       
  225:  152:-       | 152:-       | 152:-       | 148:mulq    | 148:-       
  226:  152:-       | 152:-       | 152:-       | 152:-       | 148:mulq    r3=0  
  227:  152:mulq    | 152:-       | 152:-       | 152:-       | 152:-       
  228:  156:-       | 152:mulq    | 152:-       | 152:-       | 152:-       
  229:  156:-       | 156:-       | 152:mulq    | 152:-       | 152:-       BUS_LOAD  MEM[152] accepted 1
  230:  156:-       | 156:-       | 156:-       | 152:mulq    | 152:-       
  231:  156:-       | 156:-       | 156:-       | 156:-       | 152:mulq    r3=0  
  232:  156:-       | 156:-       | 156:-       | 156:-       | 156:-       
  233:  156:-       | 156:-       | 156:-       | 156:-       | 156:-       
  234:  156:mulq    | 156:-       | 156:-       | 156:-       | 156:-       
  235:  160:-       | 156:mulq    | 156:-       | 156:-       | 156:-       
  236:  160:-       | 160:-       | 156:mulq    | 156:-       | 156:-       
  237:  160:-       | 160:-       | 160:-       | 156:mulq    | 156:-       
  238:  160:-       | 160:-       | 160:-       | 160:-       | 156:mulq    r3=0  
  239:  160:mulq    | 160:-       | 160:-       | 160:-       | 160:-       
  240:  164:-       | 160:mulq    | 160:-       | 160:-       | 160:-       
  241:  164:-       | 164:-       | 160:mulq    | 160:-       | 160:-       BUS_LOAD  MEM[160] accepted 1
  242:  164:-       | 164:-       | 164:-       | 160:mulq    | 160:-       
  243:  164:-       | 164:-       | 164:-       | 164:-       | 160:mulq    r3=0  
  244:  164:-       | 164:-       | 164:-       | 164:-       | 164:-       
  245:  164:-       | 164:-       | 164:-       | 164:-       | 164:-       
  246:  164:mulq    | 164:-       | 164:-       | 164:-       | 164:-       
  247:  168:-       | 164:mulq    | 164:-       | 164:-       | 164:-       
  248:  168:-       | 168:-       | 164:mulq    | 164:-       | 164:-       
  249:  168:-       | 168:-       | 168:-       | 164:mulq    | 164:-       
  250:  168:-       | 168:-       | 168:-       | 168:-       | 164:mulq    r3=0  
  251:  168:mulq    | 168:-       | 168:-       | 168:-       | 168:-       
  252:  172:-       | 168:mulq    | 168:-       | 168:-       | 168:-       
  253:  172:-       | 172:-       | 168:mulq    | 168:-       | 168:-       BUS_LOAD  MEM[168] accepted 1
  254:  172:-       | 172:-       | 172:-       | 168:mulq    | 168:-       
  255:  172:-       | 172:-       | 172:-       | 172:-       | 168:mulq    r3=0  
  256:  172:-       | 172:-       | 172:-       | 172:-       | 172:-       
  257:  172:-       | 172:-       | 172:-       | 172:-       | 172:-       
  258:  172:halt    | 172:-       | 172:-       | 172:-       | 172:-       
  259:  176:-       | 172:halt    | 172:-       | 172:-       | 172:-       
  260:  176:-       | 176:-       | 172:halt    | 172:-       | 172:-       
  261:  176:-       | 176:-       | 176:-       | 172:halt    | 172:-       
  262:  176:-       | 176:-       | 176:-       | 176:-       | 172:halt    
